services/web/server/src/simcore_service_webserver/chatbot/_client.py

```diff
@@ -0,0 +1,143 @@
+"""Interface to communicate with Fogbugz API
+
+- Simple client to create cases in Fogbugz
+"""
+
+import logging
+from typing import Any, Final
+from urllib.parse import urljoin
+
+import httpx
+from aiohttp import web
+from pydantic import BaseModel, Field
+from servicelib.aiohttp import status
+from tenacity import (
+    retry,
+    retry_if_exception_type,
+    retry_if_result,
+    stop_after_attempt,
+    wait_exponential,
+)
+
+from .settings import ChatbotSettings, get_plugin_settings
+
+_logger = logging.getLogger(__name__)
+
+
+_JSON_CONTENT_TYPE = "application/json"
+
+
+class ChatResponse(BaseModel):
+    answer: str = Field(description="Answer from the chatbot")
+
+
+def _should_retry(response: httpx.Response | None) -> bool:
+    if response is None:
+        return True
+    return (
+        response.status_code >= status.HTTP_500_INTERNAL_SERVER_ERROR
+        or response.status_code == status.HTTP_429_TOO_MANY_REQUESTS
+    )
+
+
+def _chatbot_retry():
+    """Retry configuration for chatbot API calls"""
+    return retry(
+        retry=(
+            retry_if_result(_should_retry)
+            | retry_if_exception_type(
+                (
+                    httpx.ConnectError,
+                    httpx.TimeoutException,
+                    httpx.NetworkError,
+                    httpx.ProtocolError,
+                )
+            )
+        ),
+        stop=stop_after_attempt(3),
+        wait=wait_exponential(multiplier=1, min=1, max=10),
+        reraise=True,
+    )
+
+
+class ChatbotRestClient:
+    def __init__(self, base_url: str, chatbot_settings: ChatbotSettings) -> None:
+        self._client = httpx.AsyncClient()
+        self._base_url = base_url
```

Comment:
Minotr: you dont need the base_url here since it alredy is coming form the chatbot_settings, I woudl remvoe it in the constructor and just assing it below
```suggestion
    def __init__(self, chatbot_settings: ChatbotSettings) -> None:
        self._client = httpx.AsyncClient()
        self._base_url = chatbot_settings.base_url
```